tests/test_vivado.py

```diff
@@ -64,3 +64,37 @@ def test_vivado_minimal():
     compare_files(ref_dir, work_root, [
         'vivado.cmd',
     ])
+
+def test_vivado_yosys():
```

Comment:
Are you able to base this off `test_vivado` rather than the minimal test? Not only would it be a more extensive test but the code should be shorter too. In fact I wonder if you could use the [pytest parameterisation support](https://docs.pytest.org/en/latest/parametrize.html) to run `test_vivado` with the two different synth settings?

Reply:
Sure, done.